packages/evolution-common/src/services/surveyGenerator/common/defaultConditional.tsx

```diff
@@ -0,0 +1,6 @@
+import { Conditional } from '../types/inputTypes';
```

Comment:
You should add the copyright header in the files that are in the evolution-common directory